### PR TITLE
Move main to jekyll theme

### PR DESCRIPTION
**Pull request by MewenLeHo**
Move `main.html` from `ods-android` to `jekyll-theme` 
+ new version of `main.html`

### CODE REVIEW

File: _layouts/main.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+    <header class="sticky-top">
+        <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation">
+            <div class="container-xxl">
+                <div class="navbar-brand me-auto me-lg-4">
+                    <a class="stretched-link" href="#">
+                        <img src="https://c.woopic.com/logo-orange.png" width="50" height="50"
+                            alt="ODS Android - Back to Home" loading="lazy">
+                    </a>
+                    <h1 class="title">ODS Android</h1>
```
Comment (julien-deramond) [inline _layouts/main.html:123]: There are 2 occurrences of "Android" that shouldn't be in this repository

**Comment by julien-deramond** [comment]
Could you please update the description to explain how to test it locally, or link something where it is explained? I'm not yet familiar with the local environment to deploy to test it.

File: _layouts/main.html (hunk 1 above)
Comment (MewenLeHo) [reply _layouts/main.html:123]: You are right and we know it. We had some trouble figuring ot the best way to organize and separate things and that's why I asked you for a meeting about the architecture of the project 😉

**Comment by MewenLeHo** [comment]
> Could you please update the description to explain how to test it locally, or link something where it is explained? I'm not yet familiar with the local environment to deploy to test it.

Did you write the process somewhere @florentmaitre? It would indeed be helpful to share it in the PR or in a wiki for instance.

**Comment by florentmaitre** [comment]
There is an explanation of how to build and run the documentation in the `ods-android` repository: https://github.com/Orange-OpenSource/ods-android/blob/develop/DEVELOP.md#documentation

However this is specific to the `ods-android` repository and depends on how you import the Jekyll theme in the repository.  Maybe we could write a document describing how to setup a development environment for `ods-jekyll-theme`? (adding Jekyll theme as submodule, configuring the `Gemfile`, etc...) I can do it if we decide to write that kind of document.

**Comment by MewenLeHo** [comment]
> There is an explanation of how to build and run the documentation in the `ods-android` repository: https://github.com/Orange-OpenSource/ods-android/blob/develop/DEVELOP.md#documentation
> 
> However this is specific to the `ods-android` repository and depends on how you import the Jekyll theme in the repository. Maybe we could write a document describing how to setup a development environment for `ods-jekyll-theme`? (adding Jekyll theme as submodule, configuring the `Gemfile`, etc...) I can do it if we decide to write that kind of document.

IMO we need this document to allow reviewers like @julien-deramond to replicate the dev env and make some tests.

**Comment by julien-deramond** [comment]
Closing it since we have now more precise needs in https://github.com/Orange-OpenSource/ods-jekyll-theme/issues/43
Thanks for the PR anyway, it's going to be useful for the next new one :)

**Pull request closed without merging**